README.md

```diff
@@ -6,7 +6,7 @@ This package is heavily inspired by [chalk's support-color](https://github.com/c
 
 ## Install
 ```sh
-go get github.com/efekarakus/termcolor
+go get github.com/jagottsicher/termcolor
```

Comment:
Did you mean to change the location of the library?

Reply:
Sorry, man, I didn't expect you're taking care to quickly. I forked and then after chnaging the der thing in version_windows.go got this repo on pkg.go.dev and republished it. still has your credits though